jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonDebugUtils.kt

```diff
@@ -49,16 +45,28 @@ object PythonDebugUtils {
                     )
                 )
 
+                val console = PyDebugConsoleBuilder(environment.project, sdk).console as PythonDebugLanguageConsoleView
                 return PyDebugProcess(
                     session,
-                    executionResult.executionConsole,
-                    executionResult.processHandler,
+                    console,
+                    null,
                     debugHost,
                     debugPorts.first()
                 ).also {
                     it.positionConverter = PathMapper.PositionConverter(PathMapper(mappings))
-
-                    PyDebugRunner.createConsoleCommunicationAndSetupActions(environment.project, executionResult, it, session)
+                    launch {
+                        for (i in 1 until 5) {
```

Comment:
can we provide our own process handler instead of this? (should just be a `DefaultDebugProcessHandler`)

alternatively, does this need be attached to the step system process handler so the stop button works correctly?

Reply:
Makes sense, that works, changed